test/view_component/view_component_browser_test.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ViewComponentBrowserTest < ViewComponent::BrowserTestCase
+  if Rails.version.to_f >= 6.1
```

Comment:
Ideally, we'd remove this conditional since we do want to enable loading the layout for Rails <= 6.1. If possible, love to see if there is any thoughts on how to make this work in those cases.